New package: SetBuilders v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: SetBuilders
- Repository: https://github.com/grnydawn/SetBuilders.jl
- Created by: @grnydawn
- Version: v0.1.1
- Commit: 2a7350ef30a99b25c560deba5877abc2edf0880b
- Reviewed by: @grnydawn
- Reference: https://github.com/grnydawn/SetBuilders.jl/commit/2a7350ef30a99b25c560deba5877abc2edf0880b#commitcomment-136244914
- Description: Julia Predicate and Enumerated Set Package
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd63d9b1b2f48f836c6a635102d0e2b1dbc6c2f68307d950277b1199f935ceeb466c700235e9fa8b06f7fb68f2b6ec6df31ba52c6e19ad471849c34cb09e07e2e93ca5f7b041e3d628fa8b3beae1df034d0177bc5dcb133b32a94c96712cfa8ed3104cbacff954e85dc2fe00bd620d6ad0d5fc2a37edb3ddb482b4b834c35545ddf4c9ab884e36d7c8a5751a31a5f779332e81439cb385c5c38e118a134f4f023bfc67f540fcb88d1e77ec020de61d3fb95 -->

grnydawn commented:
[noblock]

I updated this package to resolve the automerge issues. However, I got the following error message when I re-register it using the Registrator:

Error while trying to register: Unexpected error in registration

How can I resolve this new issue?

goerz commented:
I'm not sure about the error. Maybe the bot is just having some kind of outage? In general, you should not change the version number when you fix the `compat` issue, though. That way, you'll update the *existing* PR.

Beyond the `compat` issues, though, I would ask that you add some documentation before registering. At the very least, the README should be expanded to explain the purpose of the package and to provide a basic usage example. You may also consider making the package name more specific, e.g., `PredicateSetBuilders`. See https://github.com/JuliaRegistries/General#what-to-do-when-asked-to-reconsiderupdate-the-package-name

grnydawn commented:
[noblock]

Thanks for the info. Now, I created another commits and pushed the commits to my master branch. But, it seems that it does not trigger the update of this PR. How can I update this PR with the additional commits?

grnydawn commented:
[noblock]

There was a bug in my compat filed of Project.toml, and it is fixed now. But the CI test is still failed due to Registry dirty error:

Error: Some registries failed to update:

goerz commented:
> How can I update this PR with the additional commits?

You just comment `@JuliaRegistrator register` on the new commit

> Error: Some registries failed to update:

That's not actually an error (just a log message with an error label). There is no need to look at the CI logs: these are for internal use by the registry maintainers. If there is something wrong with the registration, the bot will leave a comment.

grnydawn commented:
[noblock]

Happy New Year!

Per your comment, is this PR going to be merged to this General Registry or is there something that I need to do further for the registration?

goerz commented:
[noblock]

You still have to fix this: 

> The compat entry for julia is unbounded.

In your `Project.toml`, you have to change

```
julia = ">= 1.2"
```

The `>=` cannot be used in the `[compat]` section for registered packages. Also: are you really using this package with Julia 1.2? If so, you should make sure that it's being automatically tested against that version.

You probably want the LTS version

```
julia = "1.6"
```

Or maybe even something newer.

JuliaRegistrator committed:
New package: SetBuilders v0.1.1

UUID: 0ea2c469-657e-49fe-9dbc-af43455b405b
Repo: https://github.com/grnydawn/SetBuilders.jl.git
Tree: 22b0b1b430554a56d186f68fa07f9f803e08ce12

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

grnydawn commented:
[noblock]

Hi, I think that I have fixed the issues for auto-merging and saw that all checks have passed. Is there anything else that I need to do for this merging? How long does it take to merge a new package into this General Registry?
